test/e2e/tests/account/lockdown.spec.ts

```diff
@@ -86,6 +86,7 @@ describe('lockdown', function (this: Mocha.Suite) {
       {
         fixtures: new FixtureBuilder().build(),
         ganacheOptions,
+        ignoredConsoleErrors: ['Error: Could not establish connection.'],
```

Comment:
this error is expected in this test. It was a blank empty error before (see screenshot in dscription) and didn't cause the test to fail. Now this is needed, as we are logging also this error message instead of an empty string and makes the test fail